### ISSUE TITLE
define of ProtoNet

### ISSUE THREAD

**Issue by mychina75** [leohsuofnthu/Tensorflow-YOLACT#1]
Hi, 
looks like the definiation of ProtoNet() different with the pytorch version.
define in pytorch: 
'mask_proto_net': [(256, 3, {'padding': 1})] * 3 + [(None, -2, {}), (256, 3, {'padding': 1})] + [(32, 1, {})],
after upsample, there is another 256,(3,3) convolution...

**Comment by leohsuofnthu** [comment]
Hi, 
Thanks for point out the mistake. I ll update soon.

**Comment by mychina75** [comment]
Great job!
